AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -364,12 +366,14 @@ - (NSString *)retrieveAuthToken {
   return authToken;
 }
 
-- (void)removeAuthToken {
+- (BOOL)removeAuthToken {
```

Comment:
There will need to be new `Assert` or `verify` statements in the unit tests to verify the return value of `removeAccount`